### PR TITLE
feat: Implement util functions to sort scenario lists

### PR DESCRIPTION
**Pull request by COSMO-VMI**

### CODE REVIEW

File: src/HierarchicalComboBox/HierarchicalComboBox.js [hunk 1]
```diff
@@ -17,25 +16,24 @@ const HierarchicalComboBox = ({
   renderInputToolType,
   ...props
 }) => {
-  const optionsList = [];
-  getFormattedOptionsList(optionsList, values, 0, separator, maxCharLength);
 
   return (
     <Autocomplete
-        {...props}
+      id="combo-box-demo"
```
Comment (csm-thu) [inline src/HierarchicalComboBox/HierarchicalComboBox.js:22]: We should remove the "combo-box-demo" id property

**Review by csm-thu — APPROVED** [review]
LGTM

File: src/HierarchicalComboBox/HierarchicalComboBox.js (hunk 1 above)
Comment (COSMO-VMI) [reply src/HierarchicalComboBox/HierarchicalComboBox.js]: Yes, I removed it. Thanks.

**Review by jreynard-code — APPROVED** [review]
LGTM ;)

**Pull request merged**